js/apps/account-ui/src/resources/ShareTheResource.tsx

```diff
@@ -138,7 +138,7 @@ export const ShareTheResource = ({
           label={t("shareUser")}
           type="string"
           helperTextInvalid={errors.usernames?.message}
-          fieldId="users"
+          fielddata-testId="users"
```

Comment:
This doesn't seem correct. Perhaps this was accidentally replaced?
```suggestion
          fieldId="users"
```